<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EpsHasuSetup</class>
 <widget class="QDialog" name="EpsHasuSetup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="close">
   <property name="geometry">
    <rect>
     <x>863</x>
     <y>13</y>
     <width>150</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>닫기</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>1024</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="siteName_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>260</y>
     <width>100</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Server Port</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="NumEdit" name="serverPort">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>260</y>
     <width>221</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="maxLength">
    <number>6</number>
   </property>
  </widget>
  <widget class="QPushButton" name="serverSave">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>260</y>
     <width>150</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>저장</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>190</y>
     <width>100</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Server IP</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="siteName">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>330</y>
     <width>100</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Site Code</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="MyLineEdit" name="siteCode">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>330</y>
     <width>221</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="maxLength">
    <number>6</number>
   </property>
  </widget>
  <widget class="IpEdit" name="serverIp">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>190</y>
     <width>221</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_7">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>460</y>
     <width>521</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>ACK</string>
   </property>
   <widget class="QPushButton" name="btnAck">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>20</y>
      <width>111</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>변경</string>
    </property>
   </widget>
   <widget class="QLabel" name="useAck">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>181</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>ACK</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NumEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">numedit.h</header>
  </customwidget>
  <customwidget>
   <class>MyLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">mylineedit.h</header>
  </customwidget>
  <customwidget>
   <class>IpEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">ipedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
